runner/sidecar/out.go

```diff
@@ -37,7 +39,10 @@ func connectOutHTTP(sink func(context.Context, []byte) error) {
 			_, _ = w.Write([]byte(err.Error()))
 			return
 		}
-		if err := sink(r.Context(), data); err != nil {
+		if err := sink(
+			dfv1.ContextWithMeta(r.Context(), fmt.Sprintf("%s.pod.%s.%s", pod, namespace, cluster), uuid.New().String()),
```

Comment:
The `source` and `ID` are changed when they are sent to downstream?

Reply:
I'm not actually sure what we want to do with that. I know we want to be able to identify messages, for tracing purposes. I think we might want to transmit it downstream (i.e. add to meta-data).